### PR TITLE
added ConfirmDialog, ConnectorDialog (v0.0.91)

### PR DESCRIPTION
**Pull request by john-whitney**
added "ConfirmDialog" and "ConnectorDialog"
also added "LoadingButton" (from @mui/lab)

### CODE REVIEW

File: src/page/storepage/StorePageGrid.tsx [hunk 1]
```diff
@@ -51,39 +52,56 @@ function StorePageGrid({
...
             });
         }
     }, [filterValue]);
 
-    React.useEffect(() => {
+    // Initialize gridRows (i.e. GridRows recoil atom)
+    useEffect(() => {
+        if (rows) {
+            // eslint-disable-next-line no-console
+            console.log(
```
Comment (missionaware) [inline src/page/storepage/StorePageGrid.tsx:85]: remove console logging

**Review by missionaware — CHANGES_REQUESTED** [review]

File: src/forms/ConfirmDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            //         transform: 'translate(-50%, -50%)'
+            //     }
+            // }}
+        >
+            <DialogTitle
+                textAlign="left"
+                margin="0px"
+                padding="5px"
+                sx={{
+                    backgroundColor: '#31363D'
```
[the remark this answers is not in the document]
Comment (john-whitney) [reply src/forms/ConfirmDialog.tsx:90]: temporary: until theming done

File: src/forms/ConfirmDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                }}
+            >
+                <Box
+                    display="flex"
+                    justifyContent="space-between"
+                    alignItems="center"
+                >
+                    <Typography variant="h6">{title}</Typography>
+                    <IconButton edge="end" color="inherit" onClick={onClose}>
+                        <Icon faKey="close" sx={{ fontSize: '14px' }} />
```
[the remark this answers is not in the document]
Comment (john-whitney) [reply src/forms/ConfirmDialog.tsx:100]: temporary: until theming done (I have TODO comment in that code area)

File: src/page/index.ts
[the remark this answers is not in the document]
Comment (john-whitney) [reply src/page/index.ts:1]: which ones?

[the remark this answers is not in the document]
Comment (ahedgeman) [reply src/page/index.ts:1]: the ones that are internal to nui (i.e. not used directly by apps).

**Pull request merged**